src/server/DBMetadata/metadataProcessor.js

```diff
@@ -66,10 +66,7 @@ function processMetadata (translateColumnType) {
 }
 
 function tableInToRef (toRef, tblCol) {
-	return (
-		toRef.hasOwnProperty(tblCol.table_name) &&
-    toRef[tblCol.table_name].hasOwnProperty(tblCol.column_name)
-	);
+	return Object.prototype.hasOwnProperty.call(tblCol.table_name) && Object[tblCol.table_name].prototype.hasOwnProperty.call(tblCol.column_name);
```

Comment:
See my other comments on verbosity of Object.prototype.hasOwnProperty.call. Also I have no ides what the heck this is supposed to be doing: Object[tblCol.table_name].prototype.hasOwnProperty.call(tblCol.column_name) is that even valid javascript?